bin/inx-chronicle/src/api/responses.rs

```diff
@@ -34,8 +34,16 @@ pub struct InfoResponse {
 
 impl_success_response!(InfoResponse);
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct SyncDataResponse(pub SyncData);
+/// An aggregation type that represents the ranges of completed milestones and gaps.
```

Comment:
This is needed for the syncer, so relegating it to the API is no bueno

Reply:
I think this is resolved for now :trollface:

Reply:
No, we still need it in the syncer